src/components/SignUpModal/SignUpModal.tsx

```diff
@@ -66,9 +66,12 @@ type Props = {
 
 const RegisterModal: React.FC<Props> = (props: Props) => (
   <RegisterModalContainer>
-    <Loginform />
     <BackAndCloseButtonContainer>
-      <BackButton>{'<'} Back</BackButton>
+      <RootContext.Consumer>
+        {(value) => {
+          return <BackButton onClick={value.openLogInPopWindow}>{'<'} Back</BackButton>;
+        }}
```

Comment:
maybe extract this callback out to make as a function?

btw, you can use hooks to consume the reactcontext